internal/services/systemcentervirtualmachinemanager/system_center_virtual_machine_manager_virtual_network_resource.go

```diff
@@ -0,0 +1,209 @@
+package systemcentervirtualmachinemanager
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/extendedlocation/2021-08-15/customlocations"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/inventoryitems"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/virtualnetworks"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/vmmservers"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/systemcentervirtualmachinemanager/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type SystemCenterVirtualMachineManagerVirtualNetworkModel struct {
+	Name                                                   string            `tfschema:"name"`
+	Location                                               string            `tfschema:"location"`
+	ResourceGroupName                                      string            `tfschema:"resource_group_name"`
+	CustomLocationId                                       string            `tfschema:"custom_location_id"`
+	SystemCenterVirtualMachineManagerServerInventoryItemId string            `tfschema:"system_center_virtual_machine_manager_server_inventory_item_id"`
+	Tags                                                   map[string]string `tfschema:"tags"`
+}
+
+var _ sdk.Resource = SystemCenterVirtualMachineManagerVirtualNetworkResource{}
+var _ sdk.ResourceWithUpdate = SystemCenterVirtualMachineManagerVirtualNetworkResource{}
+
+type SystemCenterVirtualMachineManagerVirtualNetworkResource struct{}
+
+func (r SystemCenterVirtualMachineManagerVirtualNetworkResource) ModelObject() interface{} {
+	return &SystemCenterVirtualMachineManagerVirtualNetworkModel{}
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualNetworkResource) IDValidationFunc() pluginsdk.SchemaValidateFunc {
+	return virtualnetworks.ValidateVirtualNetworkID
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualNetworkResource) ResourceType() string {
+	return "azurerm_system_center_virtual_machine_manager_virtual_network"
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualNetworkResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.SystemCenterVirtualMachineManagerVirtualNetworkName,
+		},
+
+		"location": commonschema.Location(),
+
+		"resource_group_name": commonschema.ResourceGroupName(),
```

Comment:
same problem as https://github.com/hashicorp/terraform-provider-azurerm/pull/25429/files#r1547192264

Reply:
They can be in different resource group.